From: SeongJae Park <sj@kernel.org>
Date: Mon, 6 Jan 2025 10:45:24 -0800
Subject: [PATCH] Revert "selftests/damon/config: remove configs for DAMON
 debugfs interface selftests"

This reverts commit c1ee552eafcde6e42ba7422f98a0a26cb5f98a41.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 tools/testing/selftests/damon/config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/testing/selftests/damon/config b/tools/testing/selftests/damon/config
index a68a9fead5dc..0daf38974eb0 100644
--- a/tools/testing/selftests/damon/config
+++ b/tools/testing/selftests/damon/config
@@ -1,5 +1,6 @@
 CONFIG_DAMON=y
 CONFIG_DAMON_SYSFS=y
+CONFIG_DAMON_DBGFS=y
 CONFIG_DAMON_PADDR=y
 CONFIG_DAMON_VADDR=y
 CONFIG_DAMON_RECLAIM=y
-- 
2.39.5

